pkg/component/application/email/v0/.compogen/extra-intro.mdx

```diff
@@ -1,2 +1,4 @@
+
```

Comment:
This will need to be added to every extra content appended to the intro. Why not adding the newlines in the template instead?

Reply:
I just thought that we tend to clean the blank line by [this way](https://github.com/instill-ai/pipeline-backend/blob/main/pkg/component/tools/compogen/pkg/gen/resources/templates/readme.mdx.tmpl#L13).
So, I did not want to touch template again.

So, do you mean I just take out `-` in the template?

Reply:
modify it in another branch.